### ISSUE TITLE
Bug: some features are missing a baseline year

### ISSUE THREAD

**Issue by rviscomi** [eslint/css#86]
### Environment

Node version: v23.9.0
npm version: v10.9.2
Local ESLint version: v9.20.1 (Currently used)
Global ESLint version: Not found
Operating System: darwin 23.6.0

### Which language are you using?

stylesheet

### What did you do?

<details>
<summary>Configuration</summary>

```
import globals from "globals";
import css from "@eslint/css";


/** @type {import('eslint').Linter.Config[]} */
export default [
  {
    languageOptions: {
      globals: globals.browser
    }
  },
  {
    files: ["**/*.css"],
    plugins: {
      css,
    },
    language: "css/css",
    rules: {
      "css/require-baseline": ["warn", {
        "available": 2025
      }],
    },
  },
];

```
</details>

```css
.table {
  /* baseline/table: high */
  display: table;
}
```

### What did you expect to happen?

No warnings - `display: table` is a widely available feature

### What actually happened?

```
warning  Value 'table' of property 'display' is not a 2025 available baseline feature
```

### Link to Minimal Reproducible Example

https://github.com/rviscomi/baseline-demos/blob/main/eslint/src/table.css

### Participation

- [x] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
That's okay, we can leave it as part of #82. Good catch!
